cpp/src/parquet/column_reader.cc

```diff
@@ -391,10 +394,14 @@ std::shared_ptr<Page> SerializedPageReader::NextPage() {
         throw ParquetException("Invalid page header (negative number of values)");
       }
 
+      // Uncompress if needed
```

Comment:
nit: might be better to name the method something like the comment.

Reply:
Will do :-)